libkbfs/tlf_handle.go

```diff
@@ -522,20 +522,20 @@ func (h *TlfHandle) GetCanonicalName() CanonicalTlfName {
 	return h.name
 }
 
-func buildCanonicalPath(public bool, canonicalName CanonicalTlfName) string {
+// BuildCanonicalPath returns full path for tlf name
+func BuildCanonicalPath(public bool, tlfName CanonicalTlfName) string {
 	var folderType string
 	if public {
 		folderType = "public"
 	} else {
 		folderType = "private"
 	}
-	// TODO: Handle windows paths?
-	return fmt.Sprintf("/keybase/%s/%s", folderType, canonicalName)
+	return fmt.Sprintf("/keybase/%s/%s", folderType, tlfName)
```

Comment:
> we would convert it to windows path

You mean the GUI would convert it?  I just think it's an open issue what to do about displaying canonical path names on Windows, so it still seems like a TODO.  But maybe your new comment clarifies things.